jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -83,27 +83,16 @@ def test_moment_calculation(cubeviz_helper, spectrum1d_cube, tmpdir):
 
     assert dc[2].label == 'moment 1'
 
-    assert len(dc.links) == 10
-    assert len(dc.external_links) == 4
+    assert len(dc.links) == 22
+    assert len(dc.external_links) == 2
     # Link 3D z to 2D x and 3D y to 2D y
 
-    # Link 3:
-    # Pixel Axis 0 [z] from cube.pixel_component_ids[0]
-    # Pixel Axis 1 [x] from plugin.pixel_component_ids[1]
-    assert dc.external_links[2].cids1[0] == dc[0].pixel_component_ids[0]
-    assert dc.external_links[2].cids2[0] == dc[-1].pixel_component_ids[1]
-    # Link 4:
-    # Pixel Axis 1 [y] from cube.pixel_component_ids[1]
-    # Pixel Axis 0 [y] from plugin.pixel_component_ids[0]
-    assert dc.external_links[3].cids1[0] == dc[0].pixel_component_ids[1]
-    assert dc.external_links[3].cids2[0] == dc[-1].pixel_component_ids[0]
```

Comment:
should these tests be replaced with anything with the new linking, or are they irrelevant now?

Reply:
Those external links do not exist anymore. There is nothing to replace.

Reply:
Please see https://github.com/spacetelescope/jdaviz/pull/2009#discussion_r1116306718

Reply:
ok... so should we hold off merge until someone can dig into that and confirm?  @javerbukh?

Reply:
Probably cannot get to that today but it's a good opportunity for someone to get familiar with linking! 😉